src/app/core/campaign/campaign.service.ts

```diff
@@ -14,7 +14,15 @@ export class CampaignService {
 
   getCampaign(id: number | string): Observable<CampaignState> {
     return this.augury.follow('prx:campaign', { id, zoom: 'prx:flights' }).pipe(
-      switchMap(doc => doc.followItems('prx:flights').pipe(map(flightDocs => ({ doc, flightDocs })))),
+      switchMap(doc => doc.follow('prx:flights').pipe(map(flights => ({ doc, flights })))),
+      switchMap(({ doc, flights }: { doc: HalDoc; flights: HalDoc }) => {
```

Comment:
Checks if total is greater than count/page size and if so, selects all flights with `{per: total}` param; otherwise, uses flights embedded into campaign response.